doc/build/build.md

```diff
@@ -276,3 +276,15 @@ distribution does not include the source code of dependencies.
 
 For example, `julia-1.0.0.tar.gz` is the light source distribution for the `v1.0.0` release
 of Julia, while `julia-1.0.0-full.tar.gz` is the full source distribution.
+
+## Building Julia from source with a Git checkout of one or more stdlibs
```

Comment:
Accident?

Reply:
Ugh. Yeah I pushed to the wrong branch.

Reply:
I should be able to force-push to the previous commit.

Reply:
Should be fixed now.